libtiledbsoma/src/soma/soma_array.cc

```diff
@@ -1195,6 +1195,31 @@ bool SOMAArray::_extend_and_evolve_schema<std::string>(
     return false;
 }
 
+std::vector<uint8_t> SOMAArray::_cast_bit_to_uint8(
```

Comment:
Does this need to be a method of `SOMAArray`? It only uses its args `schema` and `array`. It could be a utils function if I read correctly ...

Reply:
Yeah it would probably be better in `util/util.h`